RAGchain/preprocess/loader/hwp_loader_rust.py

```diff
@@ -0,0 +1,41 @@
+import os
+from typing import List
+from libhwp import HWPReader
+
+from langchain.docstore.document import Document
+from langchain.document_loaders.base import BaseLoader
+
+
+class HwpLoaderRust(BaseLoader):
+    # It works any OS
+    def __init__(self, path: str, *args, **kwargs):
+        self.file_path = path
+        self.result = []
+        self.only_table = []
+
+    def load(self) -> List[Document]:
+        hwp = HWPReader(self.file_path)
+
+        for paragraph in hwp.find_all('paragraph'):
+            print(str(paragraph))
+            self.result.append(str(paragraph))
+
+        page = " ".join(self.result)
+        print(page)
+        document_list = [Document(page_content=page, metadata={"source": self.file_path})]
+        return document_list
+
+    def load_table(self) -> List[Document]:
```

Comment:
Make load_table to private. 
Support option for loading tables and load() and lazy_load() methods.